ui/mirage/handlers/sync.js

```diff
@@ -95,6 +96,33 @@ export default function (server) {
     };
   };
 
+  // client counts with upgrade history
```

Comment:
Can this be moved to the clients handler? All of those endpoints are imported on line 209 and `version-history` is more client relevant